packages/twenty-server/src/engine/metadata-modules/field-metadata/services/field-metadata-related-records.service.ts

```diff
@@ -100,13 +96,49 @@ export class FieldMetadataRelatedRecordsService {
         },
         transactionManager,
       );
+
+      await this.syncNoValueViewGroup(
+        newFieldMetadata,
+        view,
+        viewGroupRepository,
+        transactionManager,
+      );
+    }
+  }
+
+  async syncNoValueViewGroup(
+    fieldMetadata: FieldMetadataEntity,
+    view: ViewWorkspaceEntity,
+    viewGroupRepository: WorkspaceRepository<ViewGroupWorkspaceEntity>,
+    transactionManager?: EntityManager,
+  ): Promise<void> {
+    const noValueGroup = view.viewGroups.find(
+      (group) => group.fieldValue === '',
```

Comment:
out of curiosity, why did we decide that no value would be an empty string rather than a null value?

Reply:
That's something we can think of, as this field was not nullable, I go that way on the first implementation few PR's ago when we want to make it movable.
cc @charlesBochet, what do you think ?

Reply:
by project convention we don't want to have two states representing a null value.

For a string: '' and null would both represent a null string. ==> We have chosen to only rely on ''
For a JSON: {}, '', [] and null would all represent the null case. ==> We have chosen to only rely on null
For an array: [] and null ==> we use []